CHANGELOG/schema.json

```diff
@@ -16,7 +16,8 @@
         "bugfix",
         "dependency",
         "deprecation",
-        "breaking_change"
+        "breaking_change",
+        "performance"
```

Comment:
Should we also update: https://github.com/Kong/kong-ee/blob/master/CHANGELOG/changelog-md-template.lua?